### PR TITLE
added sources to bottom of page

### PR DESCRIPTION
**Pull request by brinkbrink**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -284,6 +284,47 @@ <h2 style="color:white">History of <strong class="theme_blue">handheld consoles<
...
...aQtIBYdoB5VpBA8iXEbtZXqq1B15mtmi07rpRMQHL5gR0aodo1KdVasA+txr+dIrkyR2SVfaOZy5r3vfSJcvUrb0Iy5ZAUSeHWrgB6DMX1JpzhLOwDumuTMVgMCOLoQW5eEPJpyo4aOmp1LAm94+kyxnWGtLcdC5DxRYlk7PmIoklBapRw5rsKX8hrBuzcOUqJJeZQVL0BJ5Eihg7DIBUlw9X83vBy5SVKch+Ifn3xCGcmpmGYtctagpTElKlCpCRehIY9IjNlTwQrtpxYHMRMLgVdiDW1R4Q7UkJy5aZksW1YG/OObQlhE2UhIZJFRd6tc10ib+pNvQU4CSvMqYoqWcpAKjmJsftGtA8WbZkcSTmKSBw17pIFPd74NxSqg6lnPO8cVxHiq5Ar4t+EVbLpCkYWajMhM1YD1AUasxSfAUiUnCzZsxPaTZiyggutRU1Xo5LXPmYazeHswKBWYkdQSH6UpSLJsoBOZq1D635xLZKQDtPBhQBDkpcghgQXNHHx/JAmYacQ/aTSlgXMw3FaB6sW9IZSu8/Mn4D5QxkygUKf2Abm/1lfcPSJbLES8LNWqX2syZMYggKWTlqDdy14vx2BCykF3AJzO17FyK2OvpDiZKHFSx/L8/OKVWy6AingpZiwTOztnKI7ylF9VKPo5bn/GLpWB4kE5ic1iVXHdr7TiHUqSkpBIdiAOQY8rRPGywEoalE6nksxAhdtfBuhJDsLpDUZ68+vhC/6JMDgzFsbMs0HUPWHSaLQ1P4EfKOY+UBkULqVUua8WvOBasi2FKNnrUCnMtgCaqOUpBYhr2HL4Q2xWHMyUElzQtRmHC37R1EUYaaXQKMWBoHIKhR7x9ippEwJDZXsw0DjTrFkAEbJWOJOcEhzlJcjQluQ98dRKm5HEyZxcRGY5WsXc6UvByFVk9SXOtkWNwam0ET1ZVgpo5LtqxTF0/UqkVKwhMrs02AIy6lxybiD5vcNRAPYTkfViYpKPQGjcrUaG2y+7LVqoB+rpCi4seKsDrTUjRKgw8/hSKasiBJQnpDInLAd1JCiHIZ7XcM0MsBhsiAAQwqRR6kkvUFmJINLmOykjiLCiw1BS/wAogiepQQSaqS5NqsqtLd0W5RFsXyWy26oTdhS/eFNB/GJdsv2fcuPtrpy5stGAt0Cr87C/KOfSVdPup+UQh//Z">Image 8</a></p>
```
Comment (BeartoothHiker) [inline index.html:319]: Shouldn't this be a URL to an image?

File: index.html [hunk 2]
```diff
@@ -284,6 +284,47 @@ <h2 style="color:white">History of <strong class="theme_blue">handheld consoles<
...
...W+85yJ9vmctOz7E+6Z0xTwh7eGKdmE9g7C/dgLzAM7IN6zG3WGhRg5dRmmdh/DOJM/v1vb+4Go1Vdu1FvHwUOckLwt8cWaRxtC6w8nU7FmuYZs0weGbth1YnrMYLZVnP0tA6apWNJxZpFNoYWe4HE/uZ9PVajqdje4el0+dY1cRkxO+lHRfgEiXiCYcxieZZ2h9om7cSxDpkhCRWcksk9Zl+xN14V6MSBcHXh+cWaF70Xr37vLDz5Swey5Pq2x0pXFf68P+YJ8PqUh334xIdx2R8M0HOMfow09NuOzVizSZ6Ge6LvoJG8Xaf6VRGJkgR96rJ+wKXaclhNt/cJdhyReOoX51oD2cdl1zR7j9V+aytyPSZBLtNiX86fMfwmVEpHt1ViWrAqlIUybN3+ivdBBTka5x9XA1oLIq0UYfL1Mt/WZcy1QbZbvuy9WH65TwWxbpBB+v9naYBlypuX71hKXpyC9J9Sm2hxXf5fWBMGFF+st32CP+gRnDBhnqb4UwI6vaJ/C0Pv/DXPbmRLpxm5qlhDBLl1Far5+wKxjYPWGe7hsX6eanq3+Ey96MSIei0tJ+l1z2dno4UPuM2ldBS4v+yWsDVVqCrEY/22kCIKS+9Gt3LSUinSC6fsc6HqFNxnD921vKBRVprof3dC8v36WuJRnDev3bl0qGxCxdtC5xTuuapvSJINjn2eFSImRa+ivOaaU6i4g08qqqjVIbQlGkmwnh9lf2MpwJqvNUrooQimZpT5inS0o7FCp690JARTr1pS++fv6avQxKd7z62PAiVb+sSP8qrAqCulmmbNneawMl3MtZ7wLnJldVlLRe0DGcRxgOfavslPNakYq0mrB4SMcrBiV8qyR8778hvgeINPRvT/Hrq0NqhxWEi56R/NIQkiVu8h6GEnw1VEarDXQMS3sYanTXecxN5QhJFRAJYU1WgO+1g0T2EpGG2lkFy5K+OChFGheeq6UieZ2gPexwAxVXEHtD2pmAEnZXdMfIvGHnm6rXDEpYtx7Gw8FgGG9JwHjeEgBVgRLe755wA5suts9WEHsruPGlS67t9dkKllSMaCHsGdkpsDfkWQkYe5ntL2YVNW5fEMIRs2lNN9z4jNVo6kHUX986bhAEVm+1fPNsAWFnMJ/nbOj7P94k/ge3pjprV2KaDwAAAABJRU5ErkJggg==">Image 9</a></p>
```
Comment (BeartoothHiker) [inline index.html:320]: Ditto, shouldn't this be

**Review by BeartoothHiker — COMMENTED** [review]
I think there is some Copypasta here.

File: index.html (hunk 2 above)
Comment (BeartoothHiker) [reply index.html:320]: ... a URL to an image.

Comment (brinkbrink) [reply index.html:320]: @BeartoothHiker I did these pretty quickly and thought they all linked to an image except for the one that is a source for two images. Please feel free to fix! I don't have a lot more time to work on this today.

Comment (BeartoothHiker) [reply index.html:320]: Thanks Brink.  I'm trying to see where the links come from, they must already be inline.

**Pull request closed without merging**